packages/user-operation-controller/src/utils/utils.test.ts

```diff
@@ -0,0 +1,16 @@
+import { normalizeGasValue } from './utils';
+
+describe('normalizeGasValue', () => {
```

Comment:
These shouldn't be needed if the util method becomes private since this is covered by the new `PendingUserOperationTracker` unit tests.

Reply:
Done in https://github.com/MetaMask/core/pull/4161/commits/1e28a31da243f52098c5ce4536e905848a3f04b1